example-streamer-uses/src/bin/zenoh_client.rs

```diff
@@ -67,7 +67,7 @@ async fn main() -> Result<(), UStatus> {
     if !args.endpoint.is_empty() {
         // Specify the address to listen on using IPv4
         let ipv4_endpoint =
-            EndPoint::from_str(args.endpoint.as_str()).expect("Unable to set endpoint");
+            EndPoint::from_str("tcp/127.0.0.1:7447").expect("Unable to set endpoint");
```

Comment:
We should respect if the user is choosing a different endpoint here.
```suggestion
            EndPoint::from_str(args.endpoint.as_str()).expect("Unable to set endpoint");
```

I see you've already set it as default, so if needed you can update the README.md to ensure that users are aware they should not override it if using the Docker-ized setup.

Reply:
Ahh yeah that must have also been from testing. Thanks for catching all these...